processor/src/main/java/org/mapstruct/ap/internal/util/Message.java

```diff
@@ -98,6 +98,7 @@
     RETRIEVAL_TYPE_VAR_RESULT( "Can't generate mapping method for a generic type variable target." ),
     RETRIEVAL_WILDCARD_SUPER_BOUND_SOURCE( "Can't generate mapping method for a wildcard super bound source." ),
     RETRIEVAL_WILDCARD_EXTENDS_BOUND_RESULT( "Can't generate mapping method for a wildcard extends bound result." ),
+    RETRIEVAL_CONTEXT_PARAMS_WITH_SAME_TYPE( "The types of @Context parameters must be unique." ),
```

Comment:
What does this mean? I may not have two context parameters of the same type? What's the reason for this? I don't have a strong preference, just curious about the rationale behind it.